migrations/2023-12-12-150828_community_aggregates_add_local_subscribers/up.sql

```diff
@@ -0,0 +1,70 @@
+-- Couldn't find a way to put local_subscribers right after subscribers except recreating the table.
+ALTER TABLE community_aggregates
+    ADD COLUMN local_subscribers int8 NOT NULL DEFAULT 0;
+
+-- update initial value
+UPDATE
+    community_aggregates
+SET
+    local_subscribers = (
+        SELECT
+            COUNT(*)
+        FROM
+            community_follower cf
+        WHERE
+            cf.community_id = community_aggregates.community_id
+            AND (
+                SELECT
+                    local
+                FROM
+                    person
+                WHERE
+                    person.id = cf.person_id));
+
+CREATE OR REPLACE FUNCTION community_aggregates_local_subscriber_count ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    IF (TG_OP = 'INSERT') THEN
+        UPDATE
+            community_aggregates
+        SET
+            local_subscribers = local_subscribers + 1
+        FROM
+            community c
+        WHERE
+            c.id = NEW.community_id
+            AND (
+                SELECT
+                    local
+                FROM
+                    person
+                WHERE
+                    id = NEW.person_id);
+    ELSIF (TG_OP = 'DELETE') THEN
+        UPDATE
+            community_aggregates
+        SET
+            local_subscribers = local_subscribers - 1
+        FROM
+            community c
+        WHERE
+            c.id = OLD.community_id
+            AND (
+                SELECT
+                    local
+                FROM
+                    person
+                WHERE
+                    id = OLD.person_id);
+    END IF;
+    RETURN NULL;
+END
+$$;
+
+CREATE TRIGGER community_aggregates_local_subscriber_count
```

Comment:
Since you don't have `local` information in the `community_follower` table (and have to run this for every insert anyway, rather than restrict the trigger to only when `local = true`), you might as well just remove this trigger, since its the exact same as the existing one.